takeshape/takeshape-starter-shopify-gatsby
# Upgrade to Gatsby 3

### Upgrade to Gatsby 3

### Test Plan (Steps to test):

1.  Check out this branch
1.  Ask @asprouse for credentials to the sample project 
1.  `npm install`
1.  `npm run develop`
1. Verify the site is generated correctly 

### Checklist:
